### PR TITLE
Add missing shuffleboard pages

### PR DESCRIPTION
**Pull request by Daltz333**
Solves #90

### CODE REVIEW

File: source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst [hunk 1]
```diff
@@ -0,0 +1,55 @@
+Commands and Subsystems
+=======================
+
+When using the Command-based framework Shuffleboard makes it easier to understand what the robot is doing by displaying the state of various commands and subsystems in real-time.
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst]:
```suggestion
When using the command-based framework Shuffleboard makes it easier to understand what the robot is doing by displaying the state of various commands and subsystems in real-time. 
```

File: source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
+Commands and Subsystems
+=======================
+
+When using the Command-based framework Shuffleboard makes it easier to understand what the robot is doing by displaying the state of various commands and subsystems in real-time. 
+
+Displaying Subsystems
+---------------------
+
+To see the status of a subsystem while the robot is operating in either automous or telop modes, that is what it's default command is and what command is currently using that subsystem send a subsystem instance to Shuffleboard:
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst]:
```suggestion
To see the status of a subsystem while the robot is operating in either autonomous or teleoperated modes, that is what its default command is and what command is currently using that subsystem, send a subsystem instance to Shuffleboard:
```

File: source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+   .. code-tab:: java
+
+      SmartDashboard.putData(subsystem-reference);
+
+   .. code-tab:: cpp
+
+      SmartDashboard::PutData(subsystem-pointer);
+
+Shuffleboard will display the subsystem name, the default command associated with this subsystem, and the currently running command. In this example the default command for the Elevator subsystem is called "AutonomousCommand" and it is also the current command that is using the Elevator subsystem.
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst]:
```suggestion
Shuffleboard will display the subsystem name, the default command associated with this subsystem, and the currently running command. In this example the default command for the Elevator subsystem is called `AutonomousCommand` and it is also the command that is currently using the Elevator subsystem.
```

File: source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+
+.. image:: images/commands-subsystems-1.png
+
+Subsystems in Test Mode
+-----------------------
+
+In Test mode (Test/Enabled in the driver station) subsystems maybe displayed in the LiveWindow tab with the sensors and actuators of the subsystem. This is ideal for verifying of sensors are working and seeing the values that they are returning. In addition the actuators can be operated, for example motors can be operated using sliders to set the speed and direction. For PIDSubsystems the P, I, D, and F constants are displayed along with the setpoint and an enable control. This is useful for tuning PIDSubsystems by adjusting the constants, putting in a setpoint, and enabling the embedded PIDController. The the mechanisms response can be observed. Then the parameters can be changed, the PIDController re-enabled until a reasonable set of parameters is found.
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst]:
```suggestion
In Test mode (Test/Enabled in the driver station) subsystems may be displayed in the LiveWindow tab with the sensors and actuators of the subsystem. This is ideal for verifying of sensors are working by seeing the values that they are returning. In addition, actuators can be operated. For example, motors can be operated using sliders to set their commanded speed and direction. For PIDSubsystems the P, I, D, and F constants are displayed along with the setpoint and an enable control. This is useful for tuning PIDSubsystems by adjusting the constants, putting in a setpoint, and enabling the embedded PIDController. Then the mechanism's response can be observed. This cycle (change parameters, enable, and observe) can be repeated until a reasonable set of parameters is found.
```

File: source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+
+.. image:: images/commands-subsystems-2.png
+
+More information on tuning PIDSubsystems can be found here: Testing and tuning PID loops. Using RobotBuilder will automatically generate the code to get the subsystem displayed in Test mode. The code that is necessary to have subsystems displayed is shown below where subsystem-name is a string containing the name of the subsystem:
```
Comment (AustinShalit) [inline source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst]: Broken link on `Testing and tuning PID loops`.  I think this ¶ needs some work but do not have a suggestion right now.

Comment (Daltz333) [reply source/docs/software/wpilib-tools/shuffleboard/advanced-usage/shuffleboard-commands-subsystems.rst]: Whoops. Will add a TODO

**Pull request merged**